resources/mandala_tokens.json

```diff
@@ -0,0 +1,58 @@
+[
```

Comment:
I'm not sure if this section should have been added as it duplicates the section generated by Acala.

Reply:
If you really want mandala tokens. We can manually maintain here.

Reply:
Let's do it. I updated the file with some of the Karura tokens. I know that there are more, but it's a start.

This will make it easier for people to grasp that Mandala is a test network for both Acala and Karura (I had the discussion where I had to explain this a couple of times, so it is an existing issue)